e2e_tests/integration/plan.spec.js

```diff
@@ -37,6 +37,26 @@ describe('Plan output', () => {
     const password = Cypress.config('password')
     cy.connect('neo4j', password)
   })
+  it('displays the expanded details by default and displays/hides details when clicking the plan expand/collapse buttons respectively', () => {
+    cy.executeCommand(':clear')
+    cy.executeCommand(
+      'EXPLAIN MATCH (n:Person) WHERE n.age > 18 RETURN n.name ORDER BY n.age'
+    )
+
+    // initially should be expanded
+    const el1 = cy.get('[data-testid="planSvg"]', { timeout: 10000 })
```

Comment:
I could have set a `let el` and then reassign each time but I just followed the pattern elsewhere in the file for a new `const el`